sonatype-idea.xml

```diff
@@ -22,13 +22,9 @@
   <option name="IMPORT_LAYOUT_TABLE">
```

Comment:
Changes to the formatting should also be applied to the examples in https://github.com/sonatype/codestyle/tree/master/src/main which among others helps to shake out undesired differences between the config and intented results across IDEs.

Reply:
I have updated the Imports example to group java and javax imports, but I'm happy to tweak again if we decide otherwise.
That example is also consistent with the Google style guide for imports 
a4d60d5cb809e6fdf05fcffb09a63380bd1080ee